torchvision/utils.py

```diff
@@ -121,10 +126,74 @@ def save_image(
             If a file object was used instead of a filename, this parameter should always be used.
         **kwargs: Other arguments are documented in ``make_grid``.
     """
-    from PIL import Image
     grid = make_grid(tensor, nrow=nrow, padding=padding, pad_value=pad_value,
                      normalize=normalize, range=range, scale_each=scale_each)
     # Add 0.5 after unnormalizing to [0, 255] to round to nearest integer
     ndarr = grid.mul(255).add_(0.5).clamp_(0, 255).permute(1, 2, 0).to('cpu', torch.uint8).numpy()
     im = Image.fromarray(ndarr)
     im.save(fp, format=format)
+
+
+def draw_bounding_boxes(
+    image: torch.Tensor,
+    boxes: torch.Tensor,
+    labels: torch.Tensor,
```

Comment:
Since our detection models return scores as well. I think `scores` can be handled with an extra parameter like labels, but we can set it to optional. If passed we can plot scores as well.

Do let me know if that should be added.

For the second point
If the utility is fixed and simple to use I guess it might be better. I think we should provide a useful utility that works out of the box with our detection models. Since it might be very generic to handle all the cases of user.